data.tex

```diff
@@ -1,27 +1,71 @@
-% Early Science Data Products
-% Responsible: Leanne Guy
-\section{Early Science Data Products} 
+\section{Early Science Data Products}
 \label{sec:data}
 
-Here we provide a summary of the data products that are expected to be made available as part of the \esp. 
-The definitive source for all LSST data products is the Data Products Definition Document, \citep{LSE-163}.
+All Rubin Observatory LSST data products are produced by the LSST Science Pipelines \cite{2019ASPC..523..521B,2018PASJ...70S...5B}. 
+For an introduction to the Rubin data products, see \citet{RubinDataProductsAbridged}.
+Here we provide a summary of the data products that are expected to be made available as part of the \esp.
+
+Each pre-operations data preview and survey data release will be accompanied by its own release-specific DPDD, giving e.g. the  database schema for the catalogs included in that dataset.
```

Comment:
I'm not sure if it makes sense to make DPDD a generic term as you've done in this paragraph

Reply:
release-specific data products description?